Jenkinsfile_vsphere

```diff
@@ -59,21 +59,14 @@ node ("vsphere-vpn-1") {
       sh "./configure.sh"
       sh "./build.sh"
     }
-    stage('Run Module Test') {
-      def testResultsDir = rootPath+"results"
-      sh "mkdir -p ${testResultsDir}"
-
-      def runArgs = ""
-      if (env.EXTERNAL_ENCODED_VPN) {
-        runArgs = runArgs + "openvpn --daemon --config external.ovpn && sleep 10 && "
-      }
-      runArgs = runArgs + "/root/go/bin/gotestsum --format standard-verbose --packages=${testsDir} --junitfile results/${testResultsOut} --jsonfile results/${testResultsJSON} -- -timeout=${timeout} -v ${params.TEST_CASE}"
+    tage('Run Module Test') {
```

Comment:
the "s" in "stage" got clipped 😅 `tage('Run Module Test')`

Reply:
Good catch, just addressed it.